connectivity/FEATURE_BLE/source/generic/SecurityManagerImpl.cpp

```diff
@@ -613,30 +835,38 @@ ble_error_t SecurityManager::setLinkEncryption(
     link_encryption_t encryption
 )
 {
-    if (!_db) return BLE_ERROR_INITIALIZATION_INCOMPLETE;
+    tr_info("Set %d encryption to %s", connection, to_string(encryption));
```

Comment:
The problem with logs that are only meaningful in pairs is that tracing may be broken up by other traces and becomes hard to read when the meaning of a line depends on a line potentially N lines lower.

I think each trace should be self contained.

Reply:
In the case of an unfiltered log, I think you make a good point. However, if you filter the log by trace group, the stream of traces should be clearer.